.github/ISSUE_TEMPLATE/component-consensus.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report related to consensus
```

Comment:
When a user tries to create a consensus related issue, why not just have a link to the consensus repo?